packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx

```diff
@@ -56,12 +56,12 @@ const basicChartPoints = {
 const datePoints: ILineChartPoints[] = [
   {
     data: [
-      { x: new Date('01/01/2020'), y: 30 },
-      { x: new Date('02/01/2020'), y: 50 },
-      { x: new Date('03/01/2020'), y: 30 },
-      { x: new Date('04/01/2020'), y: 50 },
-      { x: new Date('05/01/2020'), y: 30 },
-      { x: new Date('06/01/2020'), y: 50 },
+      { x: new Date('2020-01-01T00:00:00.000Z'), y: 30 },
```

Comment:
we need to use deterministic dates otherwise this will fail within other timezone. local dev machine